lighthouse-cli/test/smokehouse/a11y/expectations.js

```diff
@@ -230,8 +230,12 @@ module.exports = [
         },
       },
       'accesskeys': {
-        score: null,
-        scoreDisplayMode: 'manual',
+        score: 0,
```

Comment:
to fix the CI error, it looks like this might be
```js
score: null,
scoreDisplayMode: 'notApplicable',
```
?

But did something change with the audit since removal in #7020?